Source/WebInspectorUI/UserInterface/Views/InlineSwatch.js

```diff
@@ -251,6 +265,22 @@ WI.InlineSwatch = class InlineSwatch extends WI.Object
                 return;
             }
 
+            if (this._variableType === WI.InlineSwatch.Type.Color) {
+                let colorVariable = this._delegate?.inlineSwatchColorVariableCompute(this, value.toString());
```

Comment:
ditto above about `this._delegate?.inlineSwatchGetMatchingColorVariable?.(this, value.toString)) || null`